mmseg/datasets/pipelines/transforms.py

```diff
@@ -390,6 +390,52 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class Rerange(object):
+    """Rerange the image pixel value.
+
+    Args:
+        min_value (float or int): Minimum value of the reranged image.
+            Default: 0.
+        max_value (float or int): Maximum value of the reranged image.
+            Default: 255.
+    """
+
+    def __init__(self, min_value=0, max_value=255):
+        assert isinstance(min_value, float) or isinstance(min_value, int)
+        assert isinstance(max_value, float) or isinstance(max_value, int)
+        assert min_value < max_value
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __call__(self, results):
+        """Call function to rerange images.
+
+        Args:
+            results (dict): Result dict from loading pipeline.
+
+        Returns:
+            dict: Reranged results, 'img_rerange_cfg' key is added into
+                result dict.
+        """
+
+        img = results['img']
+        img_min_value = np.min(img)
+        img_max_value = np.max(img)
+        # rerange to [0, 1]
+        img = (img - img_min_value) / (img_max_value - img_min_value)
```

Comment:
If we are reading an image with range [0.1, 1] (no pixel is black), 0.1 will be mapped to `min_value` (default 0), which is not ideal.

Reply:
That's exactly what it's going to do.